defaults/main.yml

```diff
@@ -1,4 +1,12 @@
 ---
+# react_build_cmds:
```

Comment:
Do we need these statements?

Reply:
The commented out section is intentional, shows the full structure of the react_build_cmds variable, i.e. for instance where it might not be plausible to have defined default values for some of the values.

Reply:
Should we add a comment that states what you just said?  Should we also move it next to the stuff  at line 46?

Reply:
yeah sure.